src/data/load-relations-query-builder.js

```diff
@@ -12,6 +12,8 @@ export default class LoadRelationsQueryBuilder extends DataQueryBuilder {
     this.relationModel = RelationModel
 
     this.relationName = null
+
+    this.distinct = false
```

Comment:
this is unnecessary since the class extends `DataQueryBuilder` class, as well as the getter and the setter

Reply:
+